langlib/langlib-test/src/test/resources/test-src/statements/foreach/foreach_error_handling.bal

```diff
@@ -24,31 +24,28 @@ function testArrayForeachAndTrap() returns int {
     string[] validArray = ["2", "5", "7"];
     result = trap convertAndGetSumFromArray(validArray);
     assertTrue(result is int);
-    return <int>result;
-}
-
-function testArrayForeachAndPanic() {
-    string[] invalidArray = ["2", "waruna", "7"];
-    int result = convertAndGetSumFromArray(invalidArray);
-    // This line should not be executed.
-    panic error(ASSERTION_ERROR_REASON,
-                message = "Program should be panic before this line");
+    if (result is int) {
+        return result;
+    }
+    return -1;
```

Comment:
I think we can directly assert if `result` is `14` here, instead of returning `-1` just to cause the Java assertion to fail.

Reply:
Fixed